protocol_fee/protocol_fee.gno

```diff
@@ -26,41 +35,67 @@ func DistributeProtocolFee() {
 	tokens := GetRegisteredTokens()
 	for _, token := range tokens {
 		// default distribute protocol fee percent
-		// xGNS 50%
-		// devOps 50%
+		// devOps 30%
+		// govStaker 70%
 		// ...
 
 		balance := balanceOfByRegisterCall(token, consts.PROTOCOL_FEE_ADDR)
 		if balance > 0 {
-			toDevOps := balance * 5000 / 10000
-			toXGns := balance - toDevOps
+			toDevOps := balance * devOpsPct / 10000 // default 30%
+			toGovStaker := balance - toDevOps       // default 70%
```

Comment:
```suggestion
			toDevOps := balance * devOpsPct / 10000 // default 0%
			toGovStaker := balance - toDevOps       // default 100%
```

Reply:
Would any parts of the codebase be affected by this change?

Reply:
test cases will fail, I'll update related ones